app/initElectric.ts

```diff
@@ -0,0 +1,21 @@
+import { ElectricDatabase, electrify } from "electric-sql/browser"
+import { schema } from "src/generated/client"
+
+const config = {
+    // TODO(electric): make this configurable
```

Comment:
Implement the TODOs when we deploy Electric backend + auth, also remember to authenticate the server actions